app/javascript/crayons/Tabs/Tab.jsx

```diff
@@ -0,0 +1,28 @@
+import { h } from 'preact';
+import classNames from 'classnames/bind';
+import { ButtonNew as Button, Link } from '@crayons';
+
+export const Tab = (props) => {
+  const { className, current, elements, fitted, ...otherProps } = props;
```

Comment:
```suggestion
export const Tab = ({ className, current, elements, fitted, ...otherProps }) => {
```

Just a suggestion - we could destructure straight away here